### PR TITLE
Replace some references with nuget versions.

### PR DESCRIPTION
**Pull request by skfd**
Could not build this without some packages, so I added them from Nuget. It compiles now, but I'm not sure about TeamFoundation functionality.

`Microsoft.TeamFoundation.VersionControl.Client` is deprecated on Nuget.
`Microsoft.TeamFoundation.VersionControl.Common` does not exist on Nuget and is not needed for project to build. I removed it.

Can you test that TeamFoundation stuff is working, please?

My real goal is to build latinisation for option sets in different languages. It would be my next pull request, I think.

### CODE REVIEW

**Comment by skfd** [comment]
Nope, does not run without this assemblies.

Looks like the this assemblies exist only on PC with VS Professional, so the project can not be build on VS Community.

Investigating further...

**Comment by daryllabar** [comment]
Yes @skfd.  These dlls come with VS Professional and above and are used by the Earlybound Generator to be able to automagically check-out and add files to TFS when building, but only if the "Use TFS  to check out files" option is enabled.  Is it failing to run without these assemblies, even if this option isn't checked, or just failing to build?

Also, I've looked into using NuGet before for the Xrm Sdk dlls, but I wasn't sure what to do about all the extra stuff that it brings in (http://stackoverflow.com/questions/33590629/how-to-add-required-dlls-to-command-line-app-that-connects-to-crm)  Would you have an answer for that question?

**Comment by skfd** [comment]
For now it would be reasonable to put TFS stuff into repository. I will probably shoot it up on Nuget to ease the pain, next week.

> Also, I've looked into using NuGet before for the Xrm Sdk dlls, but I wasn't sure what to do about all the extra stuff that it brings in.

I disliked the amount of stuff they crammed into one package too. I even expressed my concern to the MS guy that does their Nuget publication, but he just said something like "We understand the problem. We will look into it".

But it's easy to mitigate. We can `.gitignore` unneeded stuff and make little script to hand pick what dll should go into final product.

In conclusion, please, disregard this pull request. I like this product, but I need transliteration feature. Prototyping as we speak. Should I do pull request with TFS dlls and build script?

**Comment by daryllabar** [comment]
OK, so disregard this pull request?  

As far as the TFS and build script request, is that just so a VS Community edition can build the project?

**Comment by skfd** [comment]
It's for Continuous integration and to help other collaborators get started. Even if a person has this dlls on his machine he will have to add them manually to the project. It's fixable with no problem.

**Comment by daryllabar** [comment]
Sounds great!
